.github/workflows/scripts/native-image-tests.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# This script executes native image tests based on the MODULE_UNDER_TEST provided.
+# If MODULE_UNDER_TEST is 'vision', for example, then it runs tests under spring-cloud-gcp-vision.
+# If it is 'vision-sample' then it runs tests under spring-cloud-gcp-samples/spring-cloud-gcp-vision-api-sample
+# and spring-cloud-gcp-samples/spring-cloud-gcp-vision-ocr-demo.
+
+# Native image support has not been added for this module yet.
+EXCLUDED_MODULES=("spring-cloud-spanner-spring-data-r2dbc")
```

Comment:
Seems unfortunate to need to define this here.

Reply:
As an alternative option, perhaps this could be passed in as an argument to the script?

Reply:
Updated the script to accept this value as an argument to make it less hidden. Let me know what you think?